js/lib/corejs-build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+npm i --force .
+modules="$(cat shim.js | grep ^require  | sed "s|require('\./modules/||g"  | sed "s/');//g" | tr "\n" ",")"
```

Comment:
I am not going to this for now, as if we remove more from the `shim.js` or change its format, I will need to redo it.
For example, if we are finally left with 5-10 items I am probably just going to hard them inside the script instead of having a file with an explanation on what is what and why ;)

Reply:
OK, though I don't see a reason why not to simplify this anyway.